### PR TITLE
Add backup functionality to react native

### PR DESCRIPTION
**Pull request by roeierez**
Draft until Greelight is fixed and we will be able to test it and pass the CI.

### CODE REVIEW

File: libs/sdk-react-native/ios/BreezSDKMapper.swift [hunk 1]
```diff
@@ -152,6 +152,13 @@ class BreezSDKMapper {
         }
     }
     
+    static func dictionaryOf(backupStatus: BackupStatus) -> [String: Any] {
```
Comment (dangeross) [inline libs/sdk-react-native/ios/BreezSDKMapper.swift:155]: I think this should be `[String: Any?]` as lastBackupTime is optional/nullable

File: libs/sdk-react-native/src/index.tsx [hunk 2]
```diff
@@ -388,6 +395,11 @@ export type UnspentTransactionOutput = {
     reservedToBlock: number
 }
 
+export type BackupStatus = {
+    backedUp: boolean
+    lastBackupTime: number
```
Comment (dangeross) [inline libs/sdk-react-native/src/index.tsx:400]: `lastBackupTime?: number`

File: libs/sdk-react-native/ios/BreezSDKMapper.swift (hunk 1 above)
Comment (roeierez) [reply libs/sdk-react-native/ios/BreezSDKMapper.swift:155]: You probably right although I didn't get any complain from the compiler and it worked.

Comment (roeierez) [reply libs/sdk-react-native/ios/BreezSDKMapper.swift]: fixed

File: libs/sdk-react-native/src/index.tsx (hunk 2 above)
Comment (roeierez) [reply libs/sdk-react-native/src/index.tsx]: fixed

**Review by dangeross — APPROVED** [review]

**Pull request merged**